src/olympia/discovery/data.py

```diff
@@ -11,47 +11,39 @@ def __init__(self, *args, **kwargs):
 
 
 # At the moment the disco pane items are hardcoded in this file in the repos,
-# which allows us to integrate in our translation workflow easily.
+# which allows us to integrate in our translation workflow easily. Add-on ids
+# are used instead of slugs to prevent any accidental replacement of a deleted
+# add-on by another.
```

Comment:
I keep being tempted to implement this, only to remember I used ids on purpose, so I added this comment.